tests/SubIssue.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2025 Changyong Um <eum6211@gmail.com>                              #
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.readthedocs.io/                                              #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+################################################################################
+
+from . import Framework
+
+
+class SubIssue(Framework.TestCase):
+    def setUp(self):
+        super().setUp()
+        self.repo = self.g.get_repo("PyGithub/PyGithub", lazy=True)
+        self.issue = self.repo.get_issue(28)
+        print("setup done")
+
+    def testListSubIssues(self):
+        """
+        Test listing sub-issues of an issue.
+        """
+        sub_issues = self.issue.get_sub_issues()
+        _sub_issues = [v for i, v in enumerate(sub_issues)]
```

Comment:
Can be simplified to
```suggestion
        sub_issues = [self.issue.get_sub_issues()]
```